admin/class-yoast-dashboard-widget.php

```diff
@@ -136,4 +159,13 @@ private function statistic_items() {
 	private function filter_items( $item ) {
 		return 0 !== $item['count'];
 	}
+
+	/**
+	 * Get the user ID
+	 *
+	 * @return integer
+	 */
+	private function get_user_id() {
```

Comment:
Why adding wrapper for existing WordPress function? This method doesn't add anything new.